indico/modules/events/templates/display/now_happening.html

```diff
@@ -0,0 +1,48 @@
+<div id="now-happening" {{ text_color }} style="margin: 10px 0;">
+    <span id="now-happening-label" style="font-weight: bold;">
+        {% trans %}Now happening:{% endtrans %}
+    </span>
+    <span id="now-happening-entries">
+        {% for entry in entries %}
+            {% set obj = entry.object %}
+            {% if entry.type == timetable_entry_type.CONTRIBUTION %}
```

Comment:
`entry.type.name == 'CONTRIBUTION'` - no need to pass the whole enum object to the template that way